sdk/core/azure-core/inc/azure/core/diagnostics/logger.hpp

```diff
@@ -38,21 +38,13 @@ namespace Azure { namespace Core { namespace Diagnostics {
       Error = 4,
     };
 
-    /**
-     * @brief A callback function to receive Azure SDK log messages.
-     *
-     * @param level The log message level.
-     * @param message The log message text.
-     */
-    typedef std::function<void(Level level, std::string const& message)> Listener;
```

Comment:
Any logger readme or docs that need updating?

Reply:
Nope - but they need to be written :)